style.css

```diff
@@ -0,0 +1,86 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap');
+
+body {
+  background-color: #e5e5e5;
+  padding: 0;
+  margin: 0;
+}
+
+.intro {
+  background-image: url(./figmaImages/images/background-shapes.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  object-fit: contain;
+  max-width: fit-content;
+  width: 100%;
+  height: 718px;
+}
+
+main {
+  background-color: #fff;
+  width: 100vw;
+  height: 100vh;
+  border-radius: 0 0 0 15%;
+}
+
+header {
+  padding-top: 16px;
+  display: flex;
+  justify-content: space-between;
+}
+
+nav {
+  padding-right: 32px;
+}
+
+.my-logo a {
+  color: #6070ff;
+  font-family: 'Poppins', arial, helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 900;
+  font-size: 18px;
+  line-height: 20px;
+  padding-left: 30px;
+  text-decoration: none;
+}
+
+.greeting {
+  color: #172b4d;
+  font-family: 'Poppins', arial, helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  margin-top: 140px;
+  padding-left: 32px;
+}
+
+.description {
+  color: #344563;
+  font-family: 'Poppins', arial, helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  padding-left: 32px;
+  padding-right: 34px;
+}
+
+.connect-link {
+  color: #7f8cff;
+  font-family: 'Poppins', arial, helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  padding-left: 34px;
+}
+
+.social-media-links {
+  display: flex;
+  align-items: center;
+  gap: 16px;
+  list-style-type: none;
+}
```

Comment:
- [ ] I noticed that the background is not correctly applied to the design 😓 Kindly fix this according to the Figma template for getting a design close to the figma template as much as possible according to the project requirements ✨🔨


Your design | Given design 
---| ---|
![image](https://github.com/gilberthappi/Portfolio_setup_and_mobile_first/assets/106119105/63cde7e9-8719-42f6-b1eb-bb6e48d7a88d) | ![image](https://github.com/gilberthappi/Portfolio_setup_and_mobile_first/assets/106119105/4c2004a5-50a1-4fd8-916c-e17d70c46b11)